README.md

```diff
@@ -30,11 +30,11 @@ Don't name arguments or flags with names that contain underscore symbols, becaus
 
 <!-- usage -->
 ```sh-session
-$ npm install -g @fluencelabs/cli
+$ npm install -location=global @fluencelabs/cli
 $ fluence COMMAND
 running command...
-$ fluence (--version)
-@fluencelabs/cli/0.0.2 linux-x64 node-v16.14.0
+$ fluence --version
+@fluencelabs/cli/0.1.3 darwin-x64 node-v16.15.1
```

Comment:
this one is auto-generated as well as the others (you can spot html comments <!-- usage --> and simmilar - everything between these tags is auto-generated. We can remove the tags if we want to but we need to decide on that)

so this exact line is just an example. It depends on the machine that you use. I see 
@fluencelabs/cli/0.1.2 linux-x64 node-v16.15.0

and the version number should be changed in package.json - that's where we should change it instead, I totally forgot to keep doing that